packages/apollos-data-prayer/src/prayer-requests/data-source.js

```diff
@@ -155,53 +152,53 @@ export default class PrayerRequest extends RockApolloDataSource {
       .find(id)
       .get();
 
-  getFromIds = (ids) =>
-    this.request().filterOneOf(ids.map((id) => `Id eq ${id}`));
+  getFromIds = async (ids) => {
+    const idsFilter = ids.map((id) => `Id eq ${id}`);
+    return this.request()
+      .filterOneOf(idsFilter)
+      .get();
+  };
 
   getSavedPrayers = async () => {
     const {
       dataSources: { Followings },
     } = this.context;
 
-    try {
-      const followedPrayersRequest = await Followings.getFollowingsForCurrentUser(
-        {
-          type: 'PrayerRequest',
-        }
-      );
+    const followedPrayersRequest = await Followings.getFollowingsForCurrentUser(
+      {
+        type: 'PrayerRequest',
+      }
+    );
 
-      const entities = await followedPrayersRequest.get();
-      if (!entities.length) return [];
+    const entities = await followedPrayersRequest.get();
+    if (!entities.length) return [];
 
-      const entityIds = entities.map((entity) => entity.entityId);
-      const prayers = await this.getFromIds(uniq(entityIds)).get();
+    const entityIds = entities.map((entity) => entity.entityId);
+    const prayers = await this.getFromIds(uniq(entityIds));
 
-      // filter out flagged prayers
-      return prayers.filter(
-        (prayer) => !prayer.flagCount || prayer.flagCount === 0
-      );
-    } catch (err) {
-      throw new Error(err);
-    }
+    // filter out flagged prayers
+    return prayers.filter(
+      (prayer) => !prayer.flagCount || prayer.flagCount === 0
+    );
   };
 
   // MUTATION increment prayed, for a request
   incrementPrayed = async (parsedId) => {
     try {
       await this.put(`PrayerRequests/Prayed/${parsedId}`, {});
-      return await this.getFromId(parsedId);
-    } catch (err) {
-      throw new Error(`Unable to increment prayed request!`);
+      return this.getFromId(parsedId);
+    } catch (e) {
```

Comment:
Haha ... `e`. 😂

Reply:
from MDN :)

![Screen Shot 2019-11-25 at 9 03 45 AM](https://user-images.githubusercontent.com/2659478/69546861-84f3c000-0f62-11ea-8f90-3fcbb9839f8f.png)

Reply:
Just because Mozilla does it doesn't make it right. FWIW, my big problem isn't with things like this. `e` is an established pattern, so I get it. It doesn't mean that someone just coming in knows what `e` is supposed to be. `err` or even `error` at least tells them what it is and it isn't that much harder to write. It doesn't improve anything by removing clarity. That's all I'm saying.